lib/order.rb

```diff
@@ -0,0 +1,78 @@
+require 'csv'
+require 'customer'
+require 'pry'
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+  
+  STATUS = [:pending, :paid, :processing, :shipped, :complete]
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+   
+    if STATUS.include?(fulfillment_status)
+      @fulfillment_status = fulfillment_status
+    else 
+      raise ArgumentError
```

Comment:
It's helpful to include a message with the bad status:

```suggestion
      raise ArgumentError.new("Bad fulfillment status: #{fulfillment_status}")
```